packages/core/src/NodeExecuteFunctions.ts

```diff
@@ -805,6 +805,22 @@ export async function getBinaryDataBuffer(
 	return BinaryDataManager.getInstance().retrieveBinaryData(binaryData);
 }
 
+/**
+ * Store an incoming IBinaryData & related buffer using the configured binary data manager.
+ *
+ * @export
+ * @param {Buffer} binaryData
+ * @param {IBinaryData} data
```

Comment:
```suggestion
 * @param {IBinaryData} data
  * @param {Buffer} binaryData
```

Aware of this - artefact of late updates - will fix if this line of thinking is all good :)